### ISSUE TITLE
Wrong parameter used in FindByNameAsync method of Microsoft.AspNetCore.Identity.EntityFrameworkCore.RoleStore 

### ISSUE THREAD

**Issue by IsmailHassani** [aspnet/Identity#990]
```
/// <summary>
        /// Finds the role who has the specified normalized name as an asynchronous operation.
        /// </summary>
        /// <param name="normalizedName">The normalized role name to look for.</param>
        /// <param name="cancellationToken">The <see cref="CancellationToken"/> used to propagate notifications that the operation should be canceled.</param>
        /// <returns>A <see cref="Task{TResult}"/> that result of the look up.</returns>
        Task<Role> IRoleStore<Role>.FindByNameAsync(string normalizedRoleName, CancellationToken cancellationToken)
        {
            cancellationToken.ThrowIfCancellationRequested();
            ThrowIfDisposed();
            return Roles.FirstOrDefaultAsync(r => r.NormalizedName == normalizedName, cancellationToken);
        }

```
should be:

```
/// <summary>
        /// Finds the role who has the specified normalized name as an asynchronous operation.
        /// </summary>
        /// <param name="normalizedName">The normalized role name to look for.</param>
        /// <param name="cancellationToken">The <see cref="CancellationToken"/> used to propagate notifications that the operation should be canceled.</param>
        /// <returns>A <see cref="Task{TResult}"/> that result of the look up.</returns>
        Task<Role> IRoleStore<Role>.FindByNameAsync(string normalizedName, CancellationToken cancellationToken)
        {
            cancellationToken.ThrowIfCancellationRequested();
            ThrowIfDisposed();
            return Roles.FirstOrDefaultAsync(r => r.NormalizedName == normalizedName, cancellationToken);
        }
```

**Comment by zchhaenngg** [comment]
<param name="normalizedName">The normalized **role** name to look for.</param>

**Comment by blowdart** [comment]
Sort of. The XML documentation is wrong, The function parameter is correct :)

**Comment by blowdart** [comment]
This is resolved in 1.1
